config/module.config.php

```diff
@@ -72,9 +72,7 @@
      | AOP will check this list to apply an AOP to selected directories only,
      | leave it empty if you want AOP to be applied to all files in the appDir
      */
-    'includePaths' => [
-        $basicDirectory . '/module'
-    ],
+    'includePaths' => [],
```

Comment:
> This needs to be removed because the module/X/config/ will cause an error parsing token by the reflection process.

If you include all module registered in the application.config the file Enumerator will walk every class. In my case >900 files. Which is not a problem - but this also includes config files /module/Application/config/module.config.php

These files can't be parsed correctly and will result in an parse error.
I tried to exclude these files using a wildcard path, but that doesn't work (e.g. /module/**/config)